Neos.Neos/Classes/Controller/Module/Management/WorkspacesController.php

```diff
@@ -535,17 +541,21 @@ protected function computeSiteChanges(Workspace $selectedWorkspace)
 
         ksort($siteChanges);
         foreach ($siteChanges as $siteKey => $site) {
-            foreach ($site['documents'] as $documentKey => $document) {
-                $liveDocumentNode = $liveContext->getNodeByIdentifier($document['documentNode']->getIdentifier());
-                $siteChanges[$siteKey]['documents'][$documentKey]['isMoved'] = $liveDocumentNode && $document['documentNode']->getPath() !== $liveDocumentNode->getPath();
-                $siteChanges[$siteKey]['documents'][$documentKey]['isNew'] = $liveDocumentNode === null;
-                foreach ($document['changes'] as $changeKey => $change) {
-                    $liveNode = $liveContext->getNodeByIdentifier($change['node']->getIdentifier());
-                    $siteChanges[$siteKey]['documents'][$documentKey]['changes'][$changeKey]['isNew'] = is_null($liveNode);
-                    $siteChanges[$siteKey]['documents'][$documentKey]['changes'][$changeKey]['isMoved'] = $liveNode && $change['node']->getPath() !== $liveNode->getPath();
+            foreach ($site['documents'] as $documentDimension => $doc) {
```

Comment:
I know it is just a container for the next iteration, but the name doc is bit short... maybe something like `documentsPerDimension`